python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -1656,16 +1657,15 @@ def convert_slice_axis(node, **kwargs):
     if not ends or ends == 'None':
         # ONNX doesn't support None for ends. Since ends=None depicts
         # length of dimension, passing dimension in this case.
-        in_shape = kwargs['in_shape'][0]
-        ends = in_shape[axes]
+        ends = sys.maxsize
```

Comment:
I rewrote conversion logic for slice_axis with the new onnx slice so you might need to rebase to my pr.
https://github.com/apache/incubator-mxnet/pull/19677

Reply:
I reverted this change so we can have it fixed from you PR